aio/aio-proxy/aio_proxy/response/format_search_results.py

```diff
@@ -100,7 +105,7 @@ def get_field(field, default=None):
     # Hide most fields if unité légale is non-diffusible
     is_non_diffusible = (
         True
-        if result_unite_legale.get("statut_diffusion_unite_legale") != "O"
+        if result_unite_legale.get("statut_diffusion_unite_legale") == "P"
```

Comment:
Why the change ? Is it for RNE only UL ?

Reply:
When `statut_diffusion_unite_legale` is None, and it almost always is for RNE siren numbers, the API hides the data otherwise thinking it is `non-diffusible`